package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "bcros-business-dashboard",
+  "private": true,
+  "type": "module",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "nuxt generate",
+    "build:local": "nuxt build",
+    "dev": "nuxt dev --port=3000",
+    "generate": "nuxt generate",
+    "lint": "eslint --cache --ext '.ts,.vue' . --max-warnings 0",
+    "lint:fix": "eslint --cache --ext '.ts,.vue' . --max-warnings 0 --fix",
+    "preview": "nuxt preview",
+    "postinstall": "nuxt prepare",
+    "test:unit": "vitest",
+    "test:e2e": "cypress open",
+    "test:e2e:firefox": "cypress run firefox",
+    "test:e2e:chrome": "cypress run chrome"
+  },
+  "dependencies": {
+    "@headlessui/vue": "^1.7.22",
+    "@iconify/json": "^2.2.214",
+    "@mdi/font": "^7.4.47",
+    "@nuxt/ui": "^2.16.0",
+    "@nuxtjs/i18n": "^8.3.1",
+    "@pinia/nuxt": "^0.5.1",
+    "@vuepic/vue-datepicker": "^8.7.0",
+    "@vueuse/core": "^10.10.0",
+    "http-status-codes": "^2.3.0",
+    "keycloak-js": "^24.0.4",
+    "launchdarkly-vue-client-sdk": "^2.2.0",
+    "lodash": "^4.17.21",
```

Comment:
what is the reason to seting it on this version, dont remember seeing this in our other json files.

Reply:
this is just 4.17.21 or higher. It is set the same way in director search and should be the latest version

Reply:
removed lodash entirely (we don't need it now)